### ISSUE TITLE
Cannot deserialize the current JSON array 

### ISSUE THREAD

**Issue by semihmasat** [TICLAB/SharpSquare#18]
Hello,

I am new arround here and i tried using SharpSquare. I tried to build a simple app by i am getting this result. ( at the end of the post )

Can you guys help me out ?

Do i need token or something ? How do i do that ?

SharpSquare sharpSquare = new SharpSquare("Client id", "Client secret");

Dictionary<string, string> parameters = new Dictionary<string, string>(); 
parameters.Add("ll", "51.508515,-0.1254872"); 
parameters.Add("categoryId", "4d4b7105d754a06374d81259"); //Food 
parameters.Add("radius", "5000"); 
parameters.Add("limit", "50");
var result = sharpSquare.SearchVenues(parameters); 

result.Wait();

ERROR :+1: 
-       Thrown: "Cannot deserialize the current JSON array (e.g. [1,2,3]) into type 'FourSquare.SharpSquare.Entities.FourSquareEntityItems`1[FourSquare.SharpSquare.Entities.Venue]' because the type requires a JSON object (e.g. {"name":"value"}) to deserialize correctly.
  To fix this error either change the JSON to a JSON object (e.g. {"name":"value"}) or change the deserialized type to an array or a type that implements a collection interface (e.g. ICollection, IList) like List<T> that can be deserialized from a JSON array. JsonArrayAttribute can also be added to the type to force it to deserialize from a JSON array.
  Path 'response.venues', line 1, position 43." (Newtonsoft.Json.JsonSerializationException)  Exception Message = "Cannot deserialize the current JSON array (e.g. [1,2,3]) into type 'FourSquare.SharpSquare.Entities.FourSquareEntityItems`1[FourSquare.SharpSquare.Entities.Venue]' because the type requires a JSON object (e.g. {\"name\":\"value\"}) to deserialize correctly.\r\nTo fix this error either change the JSON to a JSON object (e.g. {\"name\":\"value\"}) or change the deserialized type to an array or a type that implements a collection interface (e.g. ICollection, IList) like List<T> that can be deserialized from a JSON array. JsonArrayAttribute can also be added to the type to force it to deserialize from a JSON array.\r\nPath 'response.venues', line 1, position 43.", Exception Type = "Newtonsoft.Json.JsonSerializationException", Exception WinRT Data = null

**Comment by hexagradia** [comment]
I have ame issue here..
Is there any tutorial for search venues?

**Comment by aykutonen** [comment]
Yes, Github version it`s working but Nuget version had same issue.

**Comment by smhlana** [comment]
I had the same issue with the SharpSquare Nuget package.
I uninstalled it and I downloaded the SharpSquare library from https://github.com/TICLAB/SharpSquare. I opened the project in Visual Studio. It could not find Newtonsoft even though it was installed, so I uninstalled Newtonsoft and installed it again via the Nuget package manager. I then referenced the SharpSquare  library in my code and it worked.
